### DISCUSSION TITLE
Include hashed IPs for reports.

### DISCUSSION THREAD

**Discussion by sussycatgirl** [stoatchat/discussions#18] [Feature Suggestions]
Reports should include the hashed IP address of the reported user.

**Comment by wait-what** [comment]
Hashing an IP is pointless, as there's only 2^32 IPv4s, it would take basically no time to find an IP from the hash.

**Comment by wait-what** [comment]
This issue inspired me to create this masterpiece [wait-what/ip-dehash](https://github.com/wait-what/ip-dehash)
